register.php

```diff
@@ -0,0 +1,114 @@
+<html>
+<head>
+    <?php
+    include_once("env.php");
+    include_once("headers.php");
+
+    $dbconn = pg_connect(pg_connect_string)
+    or die('Could not connect: ' . pg_last_error());
+    include_once("sqls.php");
+    ?>
+
+    <title>Register</title>
+</head>
+
+<body>
+    <?php
+    include_once("navbar.php");
+    navbar(URL_INDEX);
+
+    if(isset($_POST['submit'])) {
+        $email = pg_escape_string($_POST['email']);
+        $first_name = pg_escape_string($_POST['firstname']);
+        $last_name = pg_escape_string($_POST['lastname']);
+        $password = pg_escape_string($_POST['password']);
+        $passwordconfirm = pg_escape_string($_POST['passwordConfirm']);
+
+        if($first_name == ""){
+            $error[] = 'First name is empty.';
+        } else if(strlen($first_name) < 3){
+            $error[] = 'First name is too short.';
+        }
+
+        if($last_name == ""){
+            $error[] = 'Last name is empty.';
+        } else if(strlen($last_name) < 2){
+            $error[] = 'Last name is too short.';
+        }
+
+        if($email == ""){
+            $error[] = 'Email is empty.';
+        } else if(checkAccountExist($email)) {
+            $error[] = 'Account has already been created for the email address.';
+        }
+
+        if($password == ""){
+            $error[] = 'Password is empty.';
+        }
+
+        if($password != $passwordconfirm){
+            $error[] = 'Passwords do not match.';
+        }
+
+        if (!isset($error)) {
+            $hashedpassword = password_hash($password, PASSWORD_BCRYPT);
```

Comment:
~~I think we can salt the password as well. For bonus points. Hashing + salt is the industry standard.~~

Nvm. Just saw that it is provided by default if we use password_hash.
https://secure.php.net/manual/en/function.password-hash.php